### PR TITLE
Add Conjur suite certification levels

### PR DESCRIPTION
**Pull request by izgeri**
Add the certification levels doc to the Conjur/conventions folder, and add a
link that references it and provides some context to the Conjur/README

### CODE REVIEW

File: Conjur/conventions/certification-levels.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  - No known bugs of `Critical` or `High` severity exist.
+- Feature includes automated tests to validate functional workflows with CyberArk DAP.
+  - Feature has been load-tested at expected Enterprise production loads.
+  - Performance goals for feature working with CyberArk DAP have been met and
+    confirmed by testing/usage data.
+  - Code coverage meets targets for Enterprise-level products.
+- Feature documentation is complete.
+  - Documentation for using the feature with CyberArk DAP exists on the appropriate
+    documentation site(s), and includes:
+    - Supported versions of relevant components.
```
Comment (sgnn7) [inline Conjur/conventions/certification-levels.md]: What components are being referred to here?

File: Conjur/conventions/certification-levels.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    - Known limitations.
+    - Comprehensive troubleshooting information.
+    - Updated API documentation (if relevant).
```
Comment (sgnn7) [inline Conjur/conventions/certification-levels.md]: Hmm... When would this be relevant?

File: Conjur/conventions/certification-levels.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+### Characteristics
+Characteristics of a Community Level feature:
+
+- Feature/Product has a name.
+- Feature can be demonstrated with a single use case.
+- Feature has value to our customers.
+- Feature is available in a tagged version.
+  - For CyberArk-maintained software, GitHub tags will be used. If the feature
+    is part of a larger project, it must be feature flagged off by default.
```
[the remark this answers is not in the document]
Comment (boazmichaely) [reply Conjur/conventions/certification-levels.md]: good catch.

[the remark this answers is not in the document]
Comment (izgeri) [reply Conjur/conventions/certification-levels.md]: maybe a hyphen would help? `feature-flagged off`?

File: Conjur/conventions/certification-levels.md (hunk 2 above)
Comment (izgeri) [reply Conjur/conventions/certification-levels.md]: really just for conjur

**Pull request merged**